lambdas/handlers/search_patient_details_handler.py

```diff
@@ -70,7 +92,7 @@ def lambda_handler(event, context):
     except UserNotAuthorisedException as e:
         logger.error(f"PDS not authorised patient: {str(e)}")
         return ApiGatewayResponse(
-            404, "Patient is outside of your access policy", "GET"
+            404, "Patient does not exist for given NHS number", "GET"
```

Comment:
Could this message be improved to reflect that the requestors ODS code may have prevented them seeing the record?

Reply:
Negative, that information could prevent a security risk.